sr_multi_moveit/sr_multi_moveit_config/scripts/generate_load_moveit_config.py

```diff
@@ -88,13 +88,20 @@
                                    "ompl_planning.yaml")
                 generate_ompl_planning(robot, robot_config, hand_template_path, output_path=output_path, ns_=NS)
             elif command == "kinematics":
+                kinematics_file_name = rospy.get_param("/generate_load_moveit_config/kinematics_file_name",
+                                                       "kinematics.yaml")
+                kinematics_extra_file_name = rospy.get_param("/generate_load_moveit_config/kinematics_extra_file_name",
```

Comment:
why do we need to have this extra group yaml file? can't it just be extending the kinematics one?

Reply:
I guess there is a reason for it, maybe we should have a better name than extra groups?

Reply:
it was like this before, i dont know what the original reason is but I just left it as it was

Reply:
ok